projects/plugins/protect/src/js/components/threats-list/styles.module.scss

```
@@ -28,3 +28,80 @@
 .threat-item-cta {
 	margin-top: 36px;
 }
+
+.list-title {
+	display: inline-block;
+	width: 50%;
+}
+
+.threat-header {
+	float: right;
+
+	> :last-child {
+		margin-left: calc( var( --spacing-base ) * 2 );
+	}
+}
+
+.threat-footer {
+	width: 100%;
+	display: flex;
+	justify-content: right;
+	padding-top: calc( var( --spacing-base ) * 4 );
+	border-top: 1px solid var( --jp-gray );
+
+	> :last-child {
+		margin-left: calc( var( --spacing-base ) * 2 );
+	}
+}
+
+.accordion-heading {
+	display: grid;
+	grid-template-columns: repeat( 9, 1fr );
+	background-color: white;
+	padding: calc( var( --spacing-base ) * 2 ) calc( var( --spacing-base ) * 3 );
+	border: 1px solid var( --jp-gray );
+	border-bottom: none;
```

```suggestion
	border-bottom: none;
	color: #646970;
```

"Gray / Gray 50" color from Figma designs :+1:

---

Good catch! I notice this previously and thought I had fixed it with some other styling - happy to know we'll be covering all edge cases.